### ISSUE TITLE
Update documentation to match fork

### ISSUE THREAD

**Issue by mitchhentges** [mozilla-releng/binary-transparency#3]
Right now, `transparencyscript` instructions are tied to our fork's source (`BrandonTang`'s). We should update these to work for our repository instead

**Comment by MihaiTabara** [comment]
Good catch. Also, AFAIK, this *script* is unused altogether so until we're ready to re-prioritize, it's worth to warn about this. 

Also, I don't think there's a good way to move from being a "migrated fork from X" unless Brandon switches the ownership. Alternatively, we can nuke the repo and re-clone it from scratch as a new repo.

**Comment by mitchhentges** [comment]
It's true that we can't remove the "forked from BrandonTang/binary-transparency" text from the top unless we re-clone from scratch, but I'm more worried about our source files within the repository, e.g. the README:

> Clone the repo: `git clone https://github.com/BrandonTang/binary-transparency.git`

IMHO examples like that should be updated to point to our new repository :smile:

**Comment by MihaiTabara** [comment]
> > Clone the repo: `git clone https://github.com/BrandonTang/binary-transparency.git`
> 
> IMHO examples like that should be updated to point to our new repository 😄

Makes perfect sense, good catch! 👍
